### PR TITLE
Ut 59 ron

### PR DESCRIPTION
**Pull request by ronhenson**

### CODE REVIEW

**Review by PatrickSmoothstack — COMMENTED** [review]
Looks pretty straightforward.

File: src/app/components/toolbar/toolbar.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.spacer {
+  flex: 1 1 auto;
+}
```
Comment (dereklance) [inline src/app/components/toolbar/toolbar.component.scss]: new line

File: src/app/components/login/login.component.ts [hunk 2]
```diff
@@ -26,8 +23,9 @@ export class LoginComponent implements OnInit {
   constructor(private toolbarService: ToolbarService,
               private authService: AuthService,
               private route: ActivatedRoute,
+              public router: Router,
               public fb: FormBuilder) {
-      this.token = this.route.snapshot.paramMap.get('token');
+              this.token = this.route.snapshot.paramMap.get('token');
```
Comment (dereklance) [inline src/app/components/login/login.component.ts]: we don't need this token for anything

File: src/app/components/login/login-routing.module.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import {Routes, RouterModule, Router} from '@angular/router';
+import {FlightSearchPageComponent} from '../../flight-booking/pages/flight-search-page/flight-search-page.component';
+
+const routes: Routes = [
+  {
+  path: '../booking/flight',
```
Comment (dereklance) [inline src/app/components/login/login-routing.module.ts]: why does the path start with `../`

**Review by dereklance — CHANGES_REQUESTED** [review]

File: src/app/components/login/login-routing.module.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
```
Comment (dereklance) [inline src/app/components/login/login-routing.module.ts]: why do we need this file? we already have the /booking/flight route defined in the app module.

File: src/app/components/login/login.component.ts [hunk 5]
```diff
@@ -49,12 +45,12 @@ export class LoginComponent implements OnInit {
...
     this.isPending = true;
-    this.authService.loginUser(this.loginForm.value).subscribe((res: any) => {
-      localStorage.setItem('token', res.token);
+    this.authService.loginUser(this.loginForm.value).subscribe((user: Users) => {
+      localStorage.setItem('token', user.token);
       this.isPending = false;
       this.showForm = false;
       this.showLoginMsg = true;
-      this.loginMsg = 'Welcome: ' + res.firstName + ' ' + res.lastName;
+      this.router.navigate(['../booking/flight']);
```
Comment (dereklance) [inline src/app/components/login/login.component.ts]: shouldn't this be `booking/flight` instead of `../booking/flight`?

**Review by dereklance — CHANGES_REQUESTED** [review]

**Review by dereklance — APPROVED** [review]

**Review by dereklance — APPROVED** [review]

**Pull request merged**